### DISCUSSION TITLE
Combined PR?

### DISCUSSION THREAD

**Discussion by satishbraju** [josh-project/josh#1171] [Q&A] [answered]
How does josh support combined Pull Requests across multiple Git repos?

**Comment by vlad-ivanov-name** [comment, marked as answer]
Hello @satishbraju , at the moment, there’s no support for GitHub PRs and such. However, you can clone workspaces and push them to a different remote, effectively maintaining a workspace mirror. This also works if you want to create a branch in multiple mirrors.
